config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml

```diff
@@ -139,5 +130,3 @@ spec:
          # edpm firewall, change the allowed CIDR if needed
          edpm_sshd_configure_firewall: true
          edpm_sshd_allowed_ranges: ['192.168.122.0/24']
```

Comment:
This is something that is only needed for ci/dev, so we should probably make sure it's set there, then drop it here.

Reply:
Sounds good. I'll send a ci-framework PR for this one.